ftw/testing/freezer.py

```diff
@@ -69,7 +70,11 @@ def freezed_utcnow(klass):
         curse(datetime, 'utcnow', freezed_utcnow)
 
         # Replace "time.time" function
-        new_time = mktime(self.new_now.timetuple())
+        local_tz = pytz.timezone(tzname[0])
+        if self.new_now.tzinfo is None:
+            new_time = mktime(self.new_now.timetuple())
+        else:
+            new_time = mktime((self.new_now + local_tz._utcoffset).utctimetuple())
```

Comment:
i think a quick explanation as to why this is done like this would help the future reader